docs/admin/out-of-resource.md

```diff
@@ -47,6 +62,14 @@ Each threshold is of the following form:
 * valid `eviction-signal` tokens as defined above.
 * valid `operator` tokens are `<`
 * valid `quantity` tokens must match the quantity representation used by Kubernetes
+* an eviction threshold can be expressed as a percentage if ends with `%` token.
+
+For example, if a node has `10Gi` of memory, and the desire is to induce eviction
+if available memory falls below `1Gi`, an eviction threshold can be specified as either
+of the following (but not both).
+
+* `memory.available<10%`
+* `memory.available<1Gi`
```

Comment:
Where are these specified? I assume there's a flag / config for it somewhere? What are the defaults?